src/concrete/ml/quantization/post_training.py

```diff
@@ -872,7 +872,7 @@ def _process_layer(
         """
 
         return self._calibrate_layers_activation(
-            True, quantized_op, *calibration_data, quantizers=quantizers
+            False, quantized_op, *calibration_data, quantizers=quantizers
```

Comment:
So we accept that we change that?

Reply:
I don't see any other solution

Reply:
I reverted that change. Not needed with clipping based fix.